test/powershell/Modules/Microsoft.PowerShell.Security/CmsMessage.Tests.ps1

```diff
@@ -124,9 +151,12 @@ Describe "CmsMessage cmdlets thorough tests" -Tags "Feature" {
     BeforeAll {
         if ($IsWindows)
         {
-            $certLocation = Create-TestCertificate
+            $certLocation = Create-GoodCertificate
             $certLocation | Should Not BeNullOrEmpty | Out-Null
 
+            $badCertLocation = Create-BadCertificate
+            $badCertLocation | Should Not BeNullOrEmpty | Out-Null
+
             if ($IsCoreCLR)
             {
                 # PKI module is not available for PowerShell Core, so we need to use Windows PowerShell to import the cert
```

Comment:
This test cannot be run on environments other than full Windows PowerShell SKUs. What about skipping the test if the test detects that it is running on Nano Server or IoT? Is that beyond the scope of our testing?